app/assets/stylesheets/application.scss

```diff
@@ -45,36 +44,22 @@
   }
 }
 
-table {
```

Comment:
@tijmenb it would mean rewrite quite a few bootstrap classes as placeholder selectors, which would be quite a lot of work.

Without using `@extend` we generate less CSS, which is good, but I don't think it makes any difference to content tagger (small admin app for now).

I think we should try this approach and see how far we can go with it. If it becomes painful, we can always revert to what I've done in commit 95c5d40dac4c82b27b3ecccaafd90390cb92d4c5 (more specifically https://github.com/alphagov/content-tagger/blob/95c5d40dac4c82b27b3ecccaafd90390cb92d4c5/app/assets/stylesheets/application.scss#L48